.github/workflows/aws-lambda-java-events-sdk-transformer.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will be triggered if there will be changes to 
+# aws-lambda-java-events-sdk-transformer package and it builds the package.
+
+name: Java CI aws-lambda-java-events-sdk-transformer
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+    - 'aws-lambda-java-events-sdk-transformer/**'
+  pull_request:
+    branches: [ '*' ]
+    paths:
+    - 'aws-lambda-java-events-sdk-transformer/**'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    
+    # Install base module
+    - name: Build events-sdk-transformer with Maven
+      run: mvn -B install --file aws-lambda-java-events-sdk-transformer/pom.xml
```

Comment:
I think we should also be installing dependency modules here and in other workflows (ie. `aws-lambda-java-events` in this case).

If a PR contains changes for both these libraries and includes a version bump to the base package (`aws-lambda-java-events`) and a dependent change in the consumer library (`aws-lambda-java-events-sdk-transformer`), the workflow would fail as it wouldn't be able to resolve and install the new version from remote repositories.